### ISSUE TITLE
cannot install wtfos, air unit is rooted but will not install

### ISSUE THREAD

**Issue by omenfpv** [fpv-wtf/wtfos-configurator#237]
Found "pigeon_wm150_tiny"

Creating basic directory structure...

Moving busybox into place...

Linking wget...

Installing entware (can take a couple of minutes)...                       Gets stuck here

aded1aad-8460-4cbc-a52d-feddc5a02b1a

its trying to install now

**Comment by omenfpv** [comment]
finally figured it out

**Comment by stylesuxx** [comment]
@omenfpv could you summarize what the issue was and how you resolved it in the end?

**Comment by robustini** [comment]
Same problem here with Goggles V2.
From the console I can't see anything to help, but systematically entware installs after about 15 minutes, doesn't seem normal to me.

![immagine](https://user-images.githubusercontent.com/3864712/212007603-897e9da5-9721-4f58-8f58-28f3cf142dad.png)

**Comment by omenfpv** [comment]
I just kept trying and it finally worked. Sorry theres not much more to say.
